domain/src/main/java/org/oppia/android/domain/state/StateDeck.kt

```diff
@@ -218,6 +231,19 @@ class StateDeck constructor(
       .build()
   }
 
+  private fun getCurrentNeedToRevisionState(
```

Comment:
```suggestion
  private fun getStateToRevisit(
```

Sounds more descriptive of what's happening in this function.

Reply:
Thanks for the suggestion! That makes sense.